test/evergreen.yml

```diff
@@ -514,6 +514,7 @@ functions:
         aws_key: ${aws_key}
         local_file: wt-hang-analyzer.tgz
         bucket: build_external
+        optional: true
```

Comment:
From [this](https://github.com/evergreen-ci/evergreen/wiki/Project-Commands#s3-put) Evergreen wiki page

> optional: boolean to indicate if failure to find or upload this file will result in a task failure. Not compatible with local_files_include_filter.

I feel the default value of "optional" parameter is "true", as currently failure to find the tarball won't cause a task failure. Do you have a patch build to verify this change works?

Reply:
Yep, I've provided links to before and after in a [comment on the ticket](https://jira.mongodb.org/browse/WT-8522) which look to me like `optional` defaults to false.

My assumption is that because `save wt hang analyzer core/debugger files` is in the list of `post` steps it doesn't impact the success/failure of the actual task, just results in red coloured text in the logs.